packages/daemon/test/test-endo.js

```diff
@@ -444,7 +445,13 @@ test('direct termination', async t => {
   await E(host).provideWorker('worker');
 
   const counterPath = path.join(dirname, 'test', 'counter.js');
-  await E(host).importUnsafeAndEndow('worker', counterPath, 'NONE', 'counter');
+  const counterLocation = url.pathToFileURL(counterPath).href;
```

Comment:
@dckc This test demonstrates the shift in responsibility for resolving a path to a URL from the server to the client.

Reply:
ah. good to see test coverage.

Perhaps my question wasn't phrased clearly, but since it's observable from a test, it's certainly a change in behavior, from my way of looking at things.